kernel/arch/dreamcast/kernel/init.c

```diff
@@ -292,29 +303,22 @@ void arch_exit() {
             arch_menu();
             break;
         case ARCH_EXIT_REBOOT:
-            arch_shutdown();
             arch_reboot();
             break;
     }
 }
 
 /* Called to shut down the system and return to the debug handler (if any) */
-void arch_return() {
-    /* Shut down */
-    arch_shutdown();
-
+__noreturn void arch_return() {
```

Comment:
Any way that while you're here, you can just add (void) to all of these function prototypes? I'd like to start getting that stuff fixed to be like... proper C. lol.